docs/dev-env-first-time-contributors.md

```diff
@@ -697,6 +713,22 @@ $ sudo patch --directory /usr/lib/ruby/vendor_ruby/vagrant < vagrant-plugin.patc
 patching file bundler.rb
 ```
 
+#### Lost Connectivity or power during vagrant setup
+
+If you dont see `(zulip-env)` in the command prompt of the guest machine but
+something as follows when you run `vagrant ssh` :
+
+```
+$ vagrant ssh
+Welcome to Ubuntu 14.04.1 LTS (GNU/Linux 4.4.0-31-generic x86_64)
+
+    Documentation: https://help.ubuntu.com/
+    vagrant@vagrant-base-trusty-amd64:~$
+```
+
+Check the logs `/var/log/zulip_provision.log` and try to re-provision
+your vagrant machine by `vagrant provision` and then run `vagrant ssh`.
```

Comment:
I think this should be:

Check the log `/var/log/zulip_provision.log` to see whether it completed successfully (see the example of successful provisioning in "Start the dev environment above"). If it hasn't, try to re-provision your vagrant machine by running `vagrant provision`, and then run `vagrant ssh` again; if the re-provisioning did the trick, you'll now enter the virtualenv automatically.